src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,20 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, query }) => {
+  const visibleMovies = movies.filter(
+    el => el.title.toLocaleLowerCase().includes(query.toLocaleLowerCase())
```

Comment:
```suggestion
movie => movie.title.toLocaleLowerCase().includes(query.toLocaleLowerCase())
```

Reply:
I agree with Oleg. Don't use `el` naming or something like that.
It's kind of confusing